extending-valtimo/plugin/custom-plugin-definition.md

```diff
@@ -70,3 +72,448 @@ class TwitterPluginFactory(
     }
 }
 ```
+
+## Front-end
+
+To develop your own front-end plugin, the library `@valtimo/plugin` provides several interfaces which your own front-end
```

Comment:
So I'm not going to go over every instance, but in general: try to avoid talking to people directly. So this part can be rephrased to:

In order to create a custom front-end plugin, the `@valtimo/plugin` library provides several interfaces which plugins must conform to before they can be used in an implementation.